Docker/supervisord/adaguc-pgbouncer.conf

```diff
@@ -0,0 +1,16 @@
+[supervisord]
+nodaemon=true
+logfile=/dev/null
```

Comment:
We silence the logging of supervisord?

Reply:
Removing this creates a `supervisord.log` in `/adaguc/adaguc-server-master`. It contains:
```
cat supervisord.log
2024-03-06 13:22:04,597 INFO supervisord started with pid 12
2024-03-06 13:22:05,599 INFO spawned: 'adaguc' with pid 13
2024-03-06 13:22:05,600 INFO spawned: 'pgbouncer' with pid 14
2024-03-06 13:22:07,473 INFO success: adaguc entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-03-06 13:22:07,473 INFO success: pgbouncer entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
```

However when the option `logfile=/dev/null` is set, the same logging still appears through `docker logs`. Writing to a file also keeps the logging visible through `docker logs`.

I could change it so it writes to a separate log file, although I suspect we would only be able to reach this files when working/debugging locally. What do you suggest?

Reply:
If it's in the docker logs, it is fine.